lib/dpl/provider/elastic_beanstalk.rb

```diff
@@ -39,7 +39,11 @@ def push_app
         create_bucket unless bucket_exists?
 
         if options[:zip_file]
-          zip_file = File.join(Dir.pwd, options[:zip_file])
+          if File.dirname(options[:zip_file]) == "."
```

Comment:
Testing on this condition might be correct, but does seem obscure to me.

For our purposes, something like this is far more straightforward:

```ruby
zip_file = File.absolute_path(File.expand_path(options[:zip_file]))
```